quickstarts/taskqueues-backup-images/functions/index.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for t`he specific language governing permissions and
+ * limitations under the License.
+ */
+// [START import]
+"use strict";
+const path = require("path");
+const fetch = require("node-fetch");
+const functions = require("firebase-functions");
```

Comment:
```suggestion
const functions = require("firebase-functions/v2");
```

Reply:
Btw, v2 also supports modular imports so maybe we should demonstrate that too

Reply:
TQ functions works slightly better w/ v1 functions for now, as long as v2 functions don't offer deterministic url.